src/main/java/com/agency04/sbss/pizza/rest/PizzaRestController.java

```diff
@@ -0,0 +1,86 @@
+package com.agency04.sbss.pizza.rest;
+
+import com.agency04.sbss.pizza.model.*;
+import com.agency04.sbss.pizza.repo.Repository;
+import com.agency04.sbss.pizza.service.PizzaDeliveryService;
+import com.agency04.sbss.pizza.service.PizzeriaService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+@RestController
+@RequestMapping("/api")
+public class PizzaRestController {
+
+    @Autowired
+    private PizzaDeliveryService pizzaDeliveryService;
+
+    @GetMapping("/pizzeria")
+    public PizzeriaService getPizzeria() {
+
+        return pizzaDeliveryService.getPizzeriaService();
+    }
+
+    @GetMapping("/pizzeria/menu")
+    public MenuItem[] getMenu() {
+
+        return pizzaDeliveryService.getPizzeriaService().getMenu();
+    }
+
+    @GetMapping("/customer/{username}")
+    public Customer getCustomer(@PathVariable String username) {
+
+        return Repository.getInstance().getCustomerByUsername(username);
+    }
+
+    @PostMapping("/customer")
+    public Customer postCustomer(@RequestParam String username) {
+        return Repository.getInstance().addCustomer(username, false, 0);
+    }
+
+    @PutMapping("/customer")
+    public Customer putCustomer(@RequestParam String username, @RequestParam int orders) {
```

Comment:
Ovdje isto ne moze biti RequestParam nego RequestBody. Mozes mu poslati i samo username, ali mora biti polje unutar objekta.

Ovako pise u zadatku:

`Updates the customer passed in the body of the request if it exists.`

Ako ne postoji, vrati response status 404 i poruku.